histoDB/histodb-dataserver/pom.xml

```diff
@@ -153,11 +153,16 @@
         </dependency>
 
         <dependency>
+            <groupId>eu.itesla_project</groupId>
+            <artifactId>ucte-import</artifactId>
+            <version>${ipstentsoe.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Ok, the runtime scope seems to work:  the dependencies are added to the big jar. I 'm declaring the scope for the two importers implementation in the histodb-dataserver's pom.